src/InvokePlaster.ps1

```diff
@@ -1,4 +1,4 @@
-<#
+﻿<#
```

Comment:
Oh, and if you want to do the merge, go for it.  Just do it as a `Squash and merge` to clean up the commit history.